src/app/classes/diagram/run.ts

```diff
@@ -6,6 +7,7 @@ export interface Run {
     arcs: Arc[];
     elements: Element[];
     warnings: string[];
+    offset?: Coordinates;
```

Comment:
Wenn du hier die `Coordinates` importierst, sollten diese entweder hier in der Datei mit definiert sein, oder aber in einer eigenen Datei. Wenn du in der grundsätzlichen Datenklasse etwas aus einer speziellen Komponente importierst, ist das nicht so gut

Reply:
Danke Dir, die Änderungen sind im aktualisierten PR